### DISCUSSION TITLE
fisher: cannot add github.com/patrickf3139/fzf.fish -- is this a valid package?

### DISCUSSION THREAD

**Discussion by franciscolourenco** [PatrickF1/fzf.fish#95] [Q&A] [answered]
I'm getting this error message when I try to install fzf.fish
```
$ fisher add patrickf3139/fzf.fish
fisher: cannot add github.com/patrickf3139/fzf.fish -- is this a valid package?
```

fish, version 3.1.2
fisher version 3.2.9 ~/.config/fish/functions/fisher.fish

**Comment by IlanCosman** [comment, marked as answer]
Your fisher version is old. The current is 3.3.2. The issue is probably that fzf.fish is now using main as its default branch, not master. I cannot reproduce this issue with the latest version of fisher.

**Comment by PatrickF1** [comment]
Hi @franciscolourenco, I agree with Ilan that you are definitely not on the latest version of fish. That error message does not exist in fisher.fish but this is the closest I could find: https://github.com/jorgebucaran/fisher/blob/c84294b06faea11db0be83cffb90053b9927c61c/fisher.fish#L319

Don't know if the problem is master/main or not though. In either case, can you try installing the latest fisher and report back if that worked.

Thanks for jumping in Ilan!

**Comment by PatrickF1** [comment]
Also, while you're here, thank you for making https://github.com/franciscolourenco/done! One of the first plugins that caught my eye and I starred when I was checking out fish.

**Comment by franciscolourenco** [comment]
Thank you, it's working now. I assumed that fisher would self-update when running `fisher` without arguments. Also good to know that people are using done! :)
